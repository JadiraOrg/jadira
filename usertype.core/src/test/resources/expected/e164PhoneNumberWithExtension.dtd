<!ELEMENT dataset (
    E164PHONENUMBERWITHEXTENSION*)>

<!ELEMENT E164PHONENUMBERWITHEXTENSION EMPTY>
<!ATTLIST E164PHONENUMBERWITHEXTENSION
    ID CDATA #REQUIRED
    E164PHONENUMBERWITHEXTENSION CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

