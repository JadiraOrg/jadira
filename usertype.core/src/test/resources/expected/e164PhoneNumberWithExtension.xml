<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "e164PhoneNumberWithExtension.dtd">
<dataset>
  <e164PhoneNumberWithExtension ID="0" E164PHONENUMBERWITHEXTENSION="+441963350474" NAME="test_0"/>
  <e164PhoneNumberWithExtension ID="1" E164PHONENUMBERWITHEXTENSION="+441963350474;ext=000032" NAME="test_1"/>
  <e164PhoneNumberWithExtension ID="2" NAME="test_2"/>
</dataset>